clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -647,21 +658,10 @@ protected synchronized void close(long timeoutMs) {
         // interrupted using wakeup) and the leave group request which have been queued, but not
         // yet sent to the broker. Wait up to close timeout for these pending requests to be processed.
         // If coordinator is not known, requests are aborted.
-        long now = time.milliseconds();
-        long endTimeMs = now + timeoutMs;
         Node coordinator = coordinator();
-        while (coordinator != null && client.pendingRequestCount(coordinator) > 0) {
-            if (Thread.currentThread().isInterrupted())
-                throw new InterruptException("Consumer close was interrupted");
```

Comment:
We check for interrupts in `ConsumerNetworkClient.poll()`, so it seemed unnecessary.